infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/wsplugins/MachineResolver.java

```diff
@@ -67,7 +66,7 @@ public InternalMachineConfig resolve() throws InfrastructureException {
     InternalMachineConfig machineConfig =
         new InternalMachineConfig(
             toServers(containerEndpoints),
-            component.getEnv().stream().collect(Collectors.toMap(Env::getName, Env::getValue)),
+            emptyMap(),
```

Comment:
I don't think I have the full picture, but I don't understand why this change is required.

Reply:
Now we propagate environment variable directly to K8s Containers objects instead of reusing deprecated `MachineConfig` and will be applied to K8s objects on Provisioning phase.